bokeh/server/django/consumers.py

```diff
@@ -126,7 +126,7 @@ class AutoloadJsConsumer(SessionConsumer):
         resources = self.resources(server_url) if resources_param != "none" else None
         bundle = bundle_for_objs_and_resources(None, resources)
 
-        render_items = [RenderItem(sessionid=session.id, elementid=element_id, use_for_title=False)]
+        render_items = [RenderItem(token=session.token, elementid=element_id, use_for_title=False)]
         bundle.add(Script(script_for_render_items(None, render_items, app_path=app_path, absolute_url=absolute_url)))
 
         js = AUTOLOAD_JS.render(bundle=bundle, elementid=element_id)
```

Comment:
@philippjfr are any additional changes needed here? (for headers?) I am really not at all familiar with the django code yet

Reply:
I'll take a look and will actually run some django to confirm it's working.